packages/suite/src/support/messages.ts

```diff
@@ -5203,6 +5231,10 @@ export default defineMessages({
         id: 'TR_CONFIRM_ON_TREZOR',
         defaultMessage: 'Confirm on Trezor',
     },
+    TR_CONFIRM_ON_DEVICE: {
+        id: 'TR_CONFIRM_ON_DEVICE',
```

Comment:
confusing key name, there is no clear difference from  TR_CONFIRM_ON_TREZOR, what about `TR_QR_MESSAGE` or `TR_QR_RECEIVE_ADDRESS_CONFIRM` or something like that?